frontend/packages/rhoas-plugin/src/topology/components/TopologyKafkaPanel.tsx

```diff
@@ -0,0 +1,163 @@
+  const { t } = useTranslation();
+  const boostrapServerHost = obj.status?.bootstrapServerHost;
+  const url = obj.status?.metadata?.cloudUI;
+
+  return (
+    <div className="co-m-pane__body">
+      <div className="row">
+        <div className="col-sm-6">
+          <ResourceSummary resource={obj} />
+        </div>
+        {boostrapServerHost && (
+          <dl className="co-m-pane__details">
+            <dt>{t('rhoas-plugin~Bootstrap Server')}</dt>
+            <dd>{boostrapServerHost}</dd>
+          </dl>
+        )}
+        {url && (
+          <dl className="co-m-pane__details">
+            <dt>{t('rhoas-plugin~URL')}</dt>
+            <dd>
+              <a href={url}>{url}</a>
+            </dd>
+          </dl>
+        )}
+      </div>
+    </div>
+  );
+};
+
+const ResourcesComponent = ({ obj }) => {
+  const serviceAccountSecretName = obj?.spec?.credentials?.serviceAccountSecretName;
+  const { namespace } = obj.metadata;
+  // const t = useTranslation();
+  const link = (
+    <ResourceLink
+      kind={referenceForModel(SecretModel)}
+      name={serviceAccountSecretName}
+      namespace={namespace}
+    />
+  );
+
+  return (
+    <ul>
+      <h3>Secret</h3>
+      <li className="list-group-item container-fluid">
+        <div className="row">
+          <span className="col-xs-12">{link}</span>
+        </div>
+      </li>
+    </ul>
+  );
+};
+
+export const ConnectedTopologyRhoasPanel: React.FC<TopologyRhoasPanelProps> = ({
+  item,
+  selectedDetailsTab,
+  onClickTab,
+}: TopologyRhoasPanelProps) => {
+  const [showAlert, setShowAlert] = React.useState(true);
+
+  // Resource
+  const akc = item?.getData().resource;
+  if (!akc) {
+    return <>No data</>;
+  }
+
+  const handleAlertFunction = () => {
+    if (showAlert) {
+      setShowAlert(false);
+    }
+  };
+
+  return (
+    <div className="overview__sidebar-pane resource-overview">
+      <div className="overview__sidebar-pane-head resource-overview__heading">
+        <h1 className="co-m-pane__heading">
+          <div className="co-m-pane__name co-resource-item">
+            {/* <ResourceIcon
+              className="co-m-resource-icon--lg co-m-resource-kafka-connection"
+              kind={KafkaConnectionModel.abbr}
+            /> */}
+            <h3>Kafka Connection</h3>
+          </div>
+        </h1>
+        {showAlert && (
+          <div className="kafka-panel-alert">
+            <Alert
+              variant="default"
+              title="Cloud Service"
+              actionClose={<AlertActionCloseButton onClick={handleAlertFunction} />}
+              isInline
+            >
+              This resource represents service that exist outside your cluster. To view details
+              about resource please go to
+              <a href="https://cloud.redhat.com/beta/application-services/openshift-streams/">
+                {' '}
+                OpenShift Streams Apache Kafka{' '}
+              </a>{' '}
```

Comment:
Do we need the external url icon for such links? Should probably also use the PF `<Button variant="link"` component.
```suggestion
              {' '}
              <a href="https://cloud.redhat.com/beta/application-services/openshift-streams/" target="_blank">
                OpenShift Streams Apache Kafka
              </a>{' '}
```